src/modules/displayList.js

```diff
@@ -0,0 +1,36 @@
+const API_URL = 'https://www.themealdb.com/api/json/v1/1/filter.php?a=British';
+const mealData = document.querySelector('.meal-cards');
+
+const displayMeal = (data) => {
+  const meal = document.createElement('div');
+  meal.classList.add('card');
+  meal.innerHTML = `
+  <div class="card-image">
+          <img src="${data.strMealThumb}"/>
+          </div>
+          <div class="card-content">
+          <div class="first-part">
+            <h2>${data.strMeal}</h2>
+            <div class='like-part likes-${data.idMeal}'>
+            <i class="fa-solid fa-heart like-icon" id='${data.idMeal}'></i> 
+            <span class='like-count ${data.idMeal}>' id='${data.idMeal}'>0 likes</span>
+            </div>
+          </div>
+            <div class="second-part">
+            <button class="sea-meal">Comment</button>
```

Comment:
- [ ] in as much as you worked on the project alone, you still need to include a pop-up for comment where a user can view and make comments as it is in the design
![Screenshot from 2023-03-16 23-27-28](https://user-images.githubusercontent.com/66097371/225768595-3fff7e26-c5c1-4cbb-afa3-7ce526a2a914.png)

Reply:
done